hastexo/migrations/0010_add_user_foreign_key.py

```diff
@@ -33,17 +15,22 @@ def backfill_learner(apps, schema_editor):
         Stack = apps.get_model("hastexo", "Stack")
         AnonymousUserId = apps.get_model("student", "AnonymousUserId")
         for stack in Stack.objects.all():
-            stack.learner = AnonymousUserId.objects.get(
+            learner = AnonymousUserId.objects.get(
```

Comment:
I believe this will throw `DoesNotExist` instead of returning None if user cannot be found

Reply:
OK, [like so](https://github.com/hastexo/hastexo-xblock/compare/4ef5ecfb389bc16f65264a4f001553e046a136eb..b50fa24bd46081c0b8de450c277db01839c65d41)? (Reference: [Django docs](https://docs.djangoproject.com/en/3.2/ref/exceptions/#objectdoesnotexist))

Reply:
Yes, that should do it.